rbd/rbd_nautilus.go

```diff
@@ -45,3 +47,21 @@ func GetImageNames(ioctx *rados.IOContext) ([]string, error) {
 	}
 	return names, nil
 }
+
+// GetCreateTimestamp returns the time the rbd image was created.
+//
+// Implements:
+//  int rbd_get_create_timestamp(rbd_image_t image, struct timespec *timestamp);
+func (image *Image) GetCreateTimestamp() (timespec Timespec, err error) {
+	if err := image.validate(imageIsOpen); err != nil {
+		return Timespec{}, err
+	}
+
+	var cts C.struct_timespec
+
+	if ret := C.rbd_get_create_timestamp(image.image, &cts); ret < 0 {
+		return Timespec{}, RBDError(ret)
```

Comment:
I'd prefer to use getError so that any future error handling refactoring would be simpler and its just more consistent IMO.